spec/system/composer/discard_draft_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+describe "Composer - discard draft modal", type: :system do
+  fab!(:current_user) { Fabricate(:admin) }
```

Comment:
Bit of a nit pick but maybe this could be current_admin instead

Reply:
I see what you are after after, but that's a convention I use everywhere and it mimics the `current_user` we have in code.